utils/src/ooinstall/openshift_ansible.py

```diff
@@ -11,30 +11,46 @@
 
 CFG = None
 
+ROLES_TO_GROUPS_MAP = {
+    'master': 'masters',
+    'node': 'nodes',
+    'storage': 'nfs'
```

Comment:
We need to take into account the etcd group here as well. Maybe ROLES_TO_GROUPS_MAP should be dict of role -> [groups] instead of role -> group?